docs/rules/unused-variable.md

```diff
@@ -28,4 +32,6 @@ Use instead:
 def foo():
     x = 1
     return x
-```
\ No newline at end of file
+```
+
+[`dummy-variable-rgx`]: ../../settings#dummy-variable-rgx
```

Comment:
Should this be an absolute link to GitHub? What happens when the user runs `ruff rule ...`? Do we show the relative link?

Reply:
I initially linked these to the GitHub README but @sbrugman suggested to instead link the mkdocs page ... which I think makes sense, since it's weird if https://beta.ruff.rs/docs/ suddenly links you to the GitHub README.

I guess the only problem is that if you now follow these links in the GitHub README these links are broken ... which could be fixed by having absolute links to https://beta.ruff.rs/docs/settings/ ... but I think it would actually be better to stop listing the rules and configuration options in the README and have them solely on the documentation website. I'd also remove the `docs/rules/*.md` files from the master branch and only generate them for `mkdocs` ... having to run `cargo dev generate-all` all the time is annoying ... but it's your call ... I can also just change it to an absolute ruff.rs URL for now.

> What happens when the user runs ruff rule ...? Do we show the relative link?

This isn't part of the `ruff rule` output at all ... since it's only added by generate-docs for the `.md` files.

Reply:
Ah right, this makes sense. Even if we leave the rules and configuration options in the README, we could just remove the checked-in versions and have the README links point to the versions in the docs?

Reply:
We can follow-up though if we want to do that -- this makes sense for now.